bower.json

```diff
@@ -23,18 +23,40 @@
   "homepage": "https://github.com/PolymerElements/iron-location",
   "ignore": [],
   "dependencies": {
-    "polymer": "Polymer/polymer#^1.0.0"
+    "polymer": "Polymer/polymer#^2.0.0-rc.1"
   },
   "devDependencies": {
     "promise-polyfill": "polymerlabs/promise-polyfill#^1.0.0",
-    "iron-component-page": "polymerelements/iron-component-page#^1.0.0",
-    "iron-flex-layout": "polymerelements/iron-flex-layout#^1.0.0",
-    "paper-input": "polymerelements/paper-input#^1.0.0",
-    "paper-slider": "polymerelements/paper-slider#^1.0.0",
-    "paper-styles": "polymerelements/paper-styles#^1.0.0",
-    "test-fixture": "polymerelements/test-fixture#^1.0.0",
-    "web-component-tester": "^4.0.0",
-    "webcomponentsjs": "webcomponents/webcomponentsjs#^0.7.0",
-    "iron-demo-helpers": "PolymerElements/iron-demo-helpers#^1.2.3"
+    "iron-flex-layout": "PolymerElements/iron-flex-layout#2.0-preview",
+    "iron-demo-helpers": "PolymerElements/iron-demo-helpers#2.0-preview",
+    "iron-component-page": "PolymerElements/iron-component-page#2.0-preview",
+    "webcomponentsjs": "webcomponents/webcomponentsjs#^1.0.0-rc.1",
+    "test-fixture": "PolymerElements/test-fixture#^3.0.0-rc.1",
+    "web-component-tester": "Polymer/web-component-tester#^6.0.0-prerelease.6",
+    "url": "webcomponents/URL#^0.5.7"
+  },
+  "variants": {
+    "1.x": {
+      "dependencies": {
+        "polymer": "Polymer/polymer#^1.0.0"
+      },
+      "devDependencies": {
+        "promise-polyfill": "polymerlabs/promise-polyfill#^1.0.0",
+        "iron-component-page": "polymerelements/iron-component-page#^1.0.0",
+        "iron-flex-layout": "polymerelements/iron-flex-layout#^1.0.0",
+        "paper-input": "polymerelements/paper-input#^1.0.0",
+        "paper-slider": "polymerelements/paper-slider#^1.0.0",
+        "paper-styles": "polymerelements/paper-styles#^1.0.0",
+        "test-fixture": "polymerelements/test-fixture#^1.0.0",
+        "web-component-tester": "^4.0.0",
+        "webcomponentsjs": "webcomponents/webcomponentsjs#^0.7.0",
+        "iron-demo-helpers": "PolymerElements/iron-demo-helpers#^1.2.3",
+        "url": "webcomponents/URL#^0.5.7"
```

Comment:
very nitty nit: `"URL": "webcomponents/URL#^0.5.7"`
(and refer to it in the imports as `src="../../URL/url.js"`)